utils/platform/homedir_darwin.go

```diff
@@ -0,0 +1,12 @@
+//go:build darwin
+// +build darwin
+
+package platform
+
+import (
+	"path/filepath"
+)
+
+func determineDefaultHomeDirectory(username string) (string, error) {
+	return filepath.Join(`/`, "Users", username), nil
```

Comment:
Why not /Users/xxx. Won't this result in //Users. You don't really need to do filepath join if you know the os

Reply:
fair enough

Reply:
making the change